packages/docs/developer-resources/dappkit/usage.md

```diff
@@ -109,5 +105,45 @@ const cUSDBalance = this.convertToContractDecimals(
 );
 
 this.setState({ cUSDBalance, isLoadingBalance: false })
-})
 ```
+
+## DAppKit vs. DAppKit-web
+
+Originally, DAppKit was designed for mobile apps in mind and did not work out-of-the-box for web DApps running in the browser of a mobile device. DAppKit-web includes workarounds for some of the typical issues that arose for folks using DAppKit to integrate their web DApps with Valora.
+
+DAppkit uses React's `Linking` library to listen for URL changes (i.e. on return to the DApp from Valora) and then receive and parse the requested information; this causes redirection to new tabs on web browsers, however, which means that the information requested via DAppkit does not make it back to the session. To get around this, DAppkit-web uses the web browser's `localStorage` to store this returned URL, which can then be accessed by the new tabs.
```

Comment:
- React's `Linking` library  => React Native's `Linking` library
- and then receive and parse => so it can receive and parse
- does not make it back to the session => does not make it back to the original session
- which can then be accessed by the new tabs => which can then be accessed by the old tab